packages/dashmate/configs/schema/configJsonSchema.js

```diff
@@ -474,8 +474,16 @@ module.exports = {
                   type: ['integer', 'null'],
                   minimum: 1,
                 },
+                ownerPublicKey: {
+                  type: ['string', 'null'],
+                  minLength: 1,
+                },
+                ownerPrivateKey: {
```

Comment:
Not sure we should store it since we have it only for the local network. Let's just print it and take it out from logs